app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+
+  <div class="container">
+    <%= link_to "Status Reporting", root_path, class: "navbar-brand" %>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
```

Comment:
Having `active` class on this means that "Reports" link in the navbar will *always* be in focus. That might be okay for now and not what you focus on.

I wanted to mention though, at some point you may want to conditionally apply the `active` class to whatever link matches the current route.